src/electron/views/device-connect-view/components/device-connect-connected-device.scss

```diff
@@ -4,6 +4,12 @@
 @import '../../../../common/styles/fonts.scss';
 
 .device-connect-connected-device {
+    .label {
+        font-weight: $fontWeightSemiBold;
+
+        margin: 24px 0 8px 0;
```

Comment:
I think in the past we've preferred the separate written-out styles (`margin-left`, etc) rather than the short form; harder to make mistakes and easier to understand at a glance that way

Reply:
updated